operations/customer-success/README.md

```diff
@@ -138,20 +138,67 @@ Below are detailed steps on how to log a Customer Reference in Salesforce.
+                      - “We are struggling to deploy our Mattermost instance.”
+                      - “We purchased 500 licenses but we have 36 active users.”
+                       Customers are downgrading their subscription but renewing.
+    
+    **“At Risk” Definition:** Customer who a) has a clear risk of 10%+ contraction (including full churn) upon renewal and less than 3 month to correct the situation or b)                                 has communicated that they will not renew. 
+    
+                     **Examples:**
+                     - The customer is unresponsive and we are within the 90 renewal period.
+                     - Customer states: “We aren’t renewing our subscription if you don't do X.
+
+#### Bi-weekly Customer Risk Meetings (Interal)
+**Purpose:** Bi-Monthly forecast call to review Early Warning, At-Risk and Churn customers and align company efforts on retaining customers and minimizing churn.
+**Who Owns the Meeting?** This meeting is coordinated and led by the (CAM) Customer Advocacy Manager
+**Meeting Frequency:** We will meet biweekly on Mondays. This is a mandatory meeting. 
+**Duration of Meeting:** 50 Minutes
+**Meeting Etiquette:** Mattermost is a “Customer Obsessed” organization. We ask to please be mindful that we are all driving to one common objective. Successful and happy customers. Be blameless. Please avoid finger pointing and keep a positive voice tone while working with the team. We ask that only one person speak at a time to avoid confusion and so everyone can add insight to help with the resolution.
+**Who Should Attend:** 
+ - Members of the MLT Team
+ - Head of Sales
+ - Customer Success Managers
+ - Customer Success Engineering
+ - Account Executives (Optional)
+ 
+**Meeting Strategy:**
+Ahead of meeting, CSMs or AEs CSM/AE escalates a technical or business issue by logging or updating a “Early Warning/At Risk” in Salesforce.
+Early Warning/At Risk Areas to be logged in Salesforce and reviewed in Looker report.
 
+#### Customer Risk Object (SFDC)
 The Customer Risk object in Salesforce was created to track At-Risk and Early Warning customers. Managing customer risk in Salesforce allows for better visibility among departments at Mattermost.
 
 To log Customer Risk in Salesforce you can watch the [Creating Customer Risk video](https://drive.google.com/drive/u/0/folders/1_nbZhAirM2-4q4oFdygSgmS1H_pAn5qm) or follow the step-by-step instructions below.
 
 #### Instructions for Logging in Salesforce
-
+**Salesforce Account Layout Customer Risk Button**
+1. Log into your Salesforce account.
+2. Type the Account Name in the search bar.
+3. Select "Log Customer Risk" button 
+4. The quick form appears in a small pop-out window. You don't need to fill out all fields. Below are the fields you need to fill out:
+   - **Key Contact:** Who is the main contact at the account. list defaults to only contacts tied to the account you are logging the risk.
```

Comment:
I've proposed a change to the way we're defining the purpose and/or scope of a given field. My suggestions are in definition format rather than question format. Please feel free to dismiss if this approach isn't suitable. If it is suitable, consider updating the remaining content on this page to match for consistency.